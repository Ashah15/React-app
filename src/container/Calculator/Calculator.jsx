/* eslint-disable react/react-in-jsx-scope */
import App from '../App/App';
import './Calculator.style.css';

const Calculator = () => (
  <div className="calculator-container">
    <h1>Lets do it!</h1>
    <App />
  </div>
);

export default Calculator;
