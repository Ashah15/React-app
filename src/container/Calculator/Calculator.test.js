/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable react/react-in-jsx-scope */
import renderer from 'react-test-renderer';
import Calculator from './Calculator';

test('renders learn react link', () => {
  const result = renderer.create(<Calculator />).toJSON();
  expect(result).toMatchSnapshot();
});
