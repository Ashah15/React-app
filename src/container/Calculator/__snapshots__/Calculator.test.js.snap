// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="calculator-container"
>
  <h1>
    Lets do it!
  </h1>
  <div
    className="app"
  >
    <div
      className="result"
    >
      0
       
       
    </div>
    <div
      className="Groups"
    >
      <div
        className="Group1"
      >
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            AC
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            +/-
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            %
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "orange",
              }
            }
            type="button"
          >
            รท
          </button>
        </div>
      </div>
      <div
        className="Group2"
      >
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            7
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            8
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            9
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "orange",
              }
            }
            type="button"
          >
            X
          </button>
        </div>
      </div>
      <div
        className="Group3"
      >
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            4
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            5
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            6
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "orange",
              }
            }
            type="button"
          >
            -
          </button>
        </div>
      </div>
      <div
        className="Group4"
      >
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            1
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            2
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            3
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "orange",
              }
            }
            type="button"
          >
            +
          </button>
        </div>
      </div>
      <div
        className="Group5"
      >
        <div
          className="zerobtn-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            0
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#EFEFEF",
              }
            }
            type="button"
          >
            .
          </button>
        </div>
        <div
          className="button-style"
        >
          <button
            className="name-buttons"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "orange",
              }
            }
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
